com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.PostProcess.cs

```diff
@@ -292,7 +291,7 @@ void BeginPostProcessFrame(CommandBuffer cmd, HDCamera camera, HDRenderPipeline
 
             // Grab physical camera settings or a default instance if it's null (should only happen
             // in rare occasions due to how HDAdditionalCameraData is added to the camera)
-            m_PhysicalCamera = camera.physicalParameters ?? m_DefaultPhysicalCamera;
+            m_PhysicalCamera = camera.physicalParameters;
```

Comment:
just to be 100% sure:
We don't modify it right ?

Before it was shared value by reference. Now it is just a copy.

Reply:
Yes we don't modify it, it's a copy. 
I kept it like this, in case a post-process pass was modifying the physicalParameters.